### DISCUSSION TITLE
Customer Group bulk prices update

### DISCUSSION THREAD

**Discussion by boostify-pro** [PrestaShop/PrestaShop#40168] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Is your feature request related to a problem?

I have a small concern, as I manage several stores with multiple customer groups and special price for every customer group, (example B2C price, B2B price, Reseller price...)

Standard prices are publicly visible, but each customer will see prices once logged in based on his customer group.

Managing this manually is very hard and time-consuming, and managing this using specific price managements tools is also not a good idea since this is not a promotional price and prices are apt to several changes sometimes per day.

### Describe the solution you'd like

I think the easiest way is to deal with this is in the same way prices are now managed by PrestaShop, so once we set product price we can also set price for other customer groups, 

Most important is the possibility to bulk update prices using a CSV file same as for standard price, for example adding columns in the file like this: 

id, price, customer_group_1_price, customer_group_2_price, customer_group_3_price etc ... and make a bulk import.

Thank You.

### Do you plan to work on this feature?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by sallemiines** [comment]
Hello @boostify-pro ,

Thank you for your suggestion. The Maintainer Team will take it into consideration for future developments. 

Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [contacting a professional](https://www.prestashop-project.org/support/) to help you create it. 

If you or anyone else is willing to [contribute this feature](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/) to the PrestaShop Project, the teams will be happy to help by ensuring that it fits well within the project. Please let us know if that is the case!

Thank you
